src/internal_events/process.rs

```diff
@@ -0,0 +1,63 @@
+use super::InternalEvent;
+use metrics::counter;
+use std::path::PathBuf;
+
+#[derive(Debug)]
+pub struct VectorStarted;
+
+impl InternalEvent for VectorStarted {
+    fn emit_logs(&self) {
+        info!(
+            target: "vector",
+            message = "Vector is starting.",
```

Comment:
```suggestion
            message = "Vector started.",
```

Reply:
Maybe `Vector has started.` so it has the scheme as other messages?

Reply:
Yep, that works as well.